src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java

```diff
@@ -151,11 +151,10 @@ public boolean getFailIfExists() {
     }
 
     void validate() {
-      if (containsMacro("paths")) {
-        return;
-      }
-      for (String path : config.getPaths()) {
```

Comment:
it's not good that it was possible to call 'config.getPaths()'. Can you also make the Config class static so that this is impossible

Reply:
Thanks for commend. Done.